rayon-core/src/lib.rs

```diff
@@ -228,7 +228,7 @@ type PanicHandler = dyn Fn(Box<dyn Any + Send>) + Send + Sync;
 type StartHandler = dyn Fn(usize) + Send + Sync;
 
 /// The type for a closure that gets invoked when a thread exits. The
-/// closure is passed the index of the thread on which is is invoked.
+/// closure is passed the index of the thread on which is invoked.
```

Comment:
This meant to be "on which **it** is invoked", just like the docs for `StartHandler`.

Reply:
Modified.

Please review again